packages/flutter/lib/src/material/bottom_app_bar.dart

```diff
@@ -290,7 +290,7 @@ class _BottomAppBarDefaultsM2 extends BottomAppBarTheme {
 // Design token database by the script:
 //   dev/tools/gen_defaults/bin/gen_defaults.dart.
 
-// Token database version: v0_158
+// Token database version: v0_162
```

Comment:
When I ran the `dart dev/tools/gen_defaults/bin/gen_defaults.dart` to generate code, the `shadowColor` below becomes to `Color? get shadowColor => Colors.transparent;` because in its template, the type is `Color?` instead of Color. Just wonder why the shadowColor type is still `Color` but only the version number is updated here?

Reply:
I'm not sure, maybe you need to update from `master`?

Reply:
Still can see the mismatch after syncing with master. Will fix it in my next token update PR!

Reply:
Thanks!